Source/hydro/riemann_nd.F90

```diff
@@ -288,7 +288,7 @@ subroutine riemanncg(ql, qr, qpd_lo, qpd_hi, &
           clsql = gcl*pl*rl
           clsqr = gcr*pr*rr
 
-          csmall = max(qaux(i,j,k3d,QCSML), qaux(i-sx,j-sy,k3d-sz,QCSML))
+          csmall = max( max(small, small * qaux(i,j,k3d,QC)) , max(small, small * qaux(i-sx,j-sy,k3d-sz,QC)) ) 
```

Comment:
This logic can be simplified, i.e.

csmall = max(small, max(small * qaux(i,j,k3d,QC), small * qaux(i-sx,j-sy,k3d-sz,QC))

Same for the two following changes.

Reply:
True! :) I will do that!